### PR TITLE
handle maximum and minimum values (enable innate Input tag attributes…

### PR DESCRIPTION
**Pull request by TonyGreenwood**
… on React Wrapper class)

DEMO:
https://gfycat.com/FlamboyantUntidyCollie

### CODE REVIEW

File: src/components/input/Input.tsx [hunk 1]
```diff
@@ -28,6 +28,8 @@ export interface IInputOwnProps {
     validateOnMount?: boolean;
     autoFocus?: boolean;
     disabledTooltip?: string;
+    minimum?: string;
+    maximum?: string;
```
Comment (GermainBergeron) [inline src/components/input/Input.tsx]: Shouldn't it be numbers?

Comment (TonyGreenwood) [reply src/components/input/Input.tsx]: Some sites aren't too explicit about it but it's mostly used as a string
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

Comment (GermainBergeron) [reply src/components/input/Input.tsx]: React accept `number | string` but number seem a better fit IMO

Comment (TonyGreenwood) [reply src/components/input/Input.tsx]: Done.

**Pull request merged**